src/main.js

```diff
@@ -16,7 +16,7 @@ document.addEventListener("DOMContentLoaded", function(event) {
 
   const modes = Object.keys(modeModeAndMimeByName);
 
-  var componentManager;
+  var componentRelay;
```

Comment:
While we don't want to completely refactor this file because it is too much of an aside, it probably wouldn't hurt to at least modernize the lines we do touch. For example using `let` here instead of `var`.

Reply:
Yeah, I will format the codebase in another PR.